wagtail/wagtailadmin/templates/wagtailadmin/pages/listing/_page_title_choose.html

```diff
@@ -5,7 +5,7 @@
 {% endcomment %}
 
 <h2>
-    {% if page.can_choose %}
+    {% if page.can_choose and not page.is_root %}
```

Comment:
This is business logic that doesn't belong in the template. The `can_choose` flag was introduced specifically to move this logic into the view, instead of cramming a load of "is this page the correct type? Does the user have permission?" checks into the template.